syntax = "proto3";

package response_models;
import "supply-model.proto";

message SupplyResponseModel {
	int32 player_id = 1;
	bool successful_apply = 2;
	optional util_models.SupplyModel supply = 3;
}
